osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -101,11 +101,12 @@ def extract_excerpts(execution_configuration, extraction_order, bbox_args, conve
                 .format(bbox_args=bbox_args, format=export_format_key)
             subprocess.check_call(extraction_command.split(' '))
 
-            if len(os.listdir(settings.RESULT_MEDIA_ROOT)) > 0:
-                for result_file_name in os.listdir(settings.RESULT_MEDIA_ROOT):
+            directory = os.path.join(settings.RESULT_MEDIA_ROOT, str(extraction_order.id) + '_' + NAME)
```

Comment:
I will rename the argument of `create_output_file` too. I think it's no clear enough too.